lib/src/conflicts.rs

```diff
@@ -460,14 +482,17 @@ fn materialize_git_style_conflict(
         output,
         ConflictMarkerLineChar::ConflictStart,
         conflict_marker_len,
-        &format!("Side #1 ({conflict_info})"),
+        &format!(
+            "Side #1 ({conflict_info}){}",
+            no_eol_marker(no_eol.get_add(0))
+        ),
     )?;
     output.write_all(left)?;
```

Comment:
I think missing `\n` should be padded here. (applies to the other materialization styles and sides)

It's the matter of rendering.